CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.9.3-py2] - 2020-01-17
+
+### Added
+
+  - Create dockerfile to build flower image
+
+[Unreleased]: https://github.com/openfun/flower-docker/compare/v0.9.3-py2...HEAD
+[0.9.3-py2]: https://github.com/openfun/flower-docker/compare/4d420c9a984230dc6c268cf0907a4e150e2e3067...v0.9.3-py2
```

Comment:
You can point this first release link to the GitHub release link instead of a comparison:
```suggestion
[0.9.3-py2]: https://github.com/openfun/flower-docker/releases/tag/v0.9.3-py2
```